roachtest: npgsql failed

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11150501?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11150501?buildTab=artifacts#/npgsql) on master @ [32f846ff670bf26f1eb09309cdc7abae8878cf39](https://github.com/cockroachdb/cockroach/commits/32f846ff670bf26f1eb09309cdc7abae8878cf39):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-32f846ff670bf26f1eb09309cdc7abae8878cf39
Tests run against npgsql v7.0.2
3297 Total Tests Run
2563 tests passed
734 tests failed
248 tests skipped
44 tests ignored
0 tests passed unexpectedly
4 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CommandTests(NonMultiplexing).Cursor_move_RecordsAffected -  (unexpected)
--- FAIL: Npgsql.Tests.CommandTests(NonMultiplexing).CloseConnection_with_exception -  (unexpected)
--- FAIL: Npgsql.Tests.CommandTests(NonMultiplexing).Use_across_connection_change(Prepared) -  (unexpected)
--- FAIL: Npgsql.Tests.CommandTests(NonMultiplexing).Cached_command_clears_parameters_placeholder_type -  (unexpected)
For a full summary look at the npgsql artifacts 
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/p/roachfana//1691001248956/1691001487301)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-30298

--- a/pkg/cmd/roachtest/tests/npgsql_blocklist.go
+++ b/pkg/cmd/roachtest/tests/npgsql_blocklist.go
@@ -745,8 +745,14 @@ var npgsqlBlocklist = blocklist{
 }
 
 var npgsqlIgnoreList = blocklist{
+	`Npgsql.Tests.CommandTests(Multiplexing).QueryNonQuery`:                                           "flaky",
+	`Npgsql.Tests.CommandTests(Multiplexing).SingleNonQuery`:                                          "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).Statement_mapped_output_parameters(Default)`:             "flaky",
+	`Npgsql.Tests.CommandTests(NonMultiplexing).Cached_command_clears_parameters_placeholder_type`:    "flaky",
+	`Npgsql.Tests.CommandTests(NonMultiplexing).CloseConnection_with_exception`:                       "flaky",
+	`Npgsql.Tests.CommandTests(NonMultiplexing).Cursor_move_RecordsAffected `:                         "flaky",
 	`Npgsql.Tests.CommandTests(NonMultiplexing).Statement_mapped_output_parameters(SequentialAccess)`: "flaky",
+	`Npgsql.Tests.CommandTests(NonMultiplexing).Use_across_connection_change(Prepared)`:               "flaky",
 	`Npgsql.Tests.ConnectionTests(NonMultiplexing).PostgreSqlVersion_ServerVersion`:                   "flaky",
 	`Npgsql.Tests.ConnectionTests(NonMultiplexing).Connector_not_initialized_exception`:               "flaky",
 	`Npgsql.Tests.ConnectionTests(NonMultiplexing).Many_open_close_with_transaction`:                  "flaky",

